CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.12.3]
+### Changed
+- update throttle for fast clicking via `isThrottledFastClicking` param and not only for press+hold
```

Comment:
Would you mind if we rephrased this a bit to make it more clear?

### Changed
- Added `keypressThrottling` parameter to allow for throttling individual keypresses